### DISCUSSION TITLE
Can you please tell me how to replace the SIC553-04 photodiode?

### DISCUSSION THREAD

**Discussion by denis1111** [srozum/film_camera_tester#2] [Q&A] [answered]
Hello. Can you please tell me how to replace the SIC553-04 photodiode? I can not find it for sale, maybe there are analogues available? Thanx

**Comment by srozum** [comment, marked as answer]
Hi, SIC553-04 is a light-to-logic converter (a photodiode combined with transimpedance  amplifier and a schmitt trigger). Also it's a photodiode used in those modules called "Laser detector" available on Amazon, AliExpress, everywhere.

**Reply by denis1111** [reply]
Thanks for the answer! Found something similar on aliexpress, is that it? :)

<img width="708" alt="111" src="https://user-images.githubusercontent.com/18027521/210049946-03724223-fdf6-4985-8f56-59f75eb5972d.png">

**Reply by srozum** [reply]
Yep, that's it.

**Reply by denis1111** [reply]
OK, thanks. I bought it, I hope I can repeat your project, I've been hatching this idea since the beginning of the year :)

**Reply by neotil1** [reply]
Hi, just letting you know I bought these exact sensors (also from Aideepen store), and they were of the incorrect type.

Quote from the item description:
`output high level when receive laser signal;
output low level when not receive laser signal.`

This results in the shutter testing the time it takes you to crank the film advance and firing your second shot instead of the speeds. Although this is great to practice speedy cranking, it's not what we're trying to achieve!

Check out the thread I opened here #12. Adding a bunch of transistors is tedious, but srozum is correct in saying that the "new" sensors on AliExpress may result in bad readings.

I've not been able to find the correct parts yet. All listings on AliExpress seem to have flipped states.

**Reply by denis1111** [reply]
Hello! Thank you for the warning! Just wanted to start assembling the gadget in the next few days. Please let me know as soon as you have information, I will follow.

**Comment by Surreal62** [comment]
One thing to be aware of is AliExpress are selling both normal and inverted photodiodes.
I received a pack of ten in December and three out the ten were inverted and the remainder normal 
There is no visual way to tell them apart, so this can cause issues when sensor building as the firmware cannot make allowances.
